village/src/components/SmurfForm.js

```diff
@@ -13,12 +14,22 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    
+    const newSmurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height
+    }
+
+    this.props.postSmurf(newSmurf);
 
     this.setState({
       name: '',
       age: '',
       height: ''
     });
+    window.location.reload();
```

Comment:
You can make the page update without using the window.location.reload and use setState to update the state again instead.